### ISSUE TITLE
Overlay viewports do not block picking world coordinate projection

### ISSUE THREAD

**Issue by mcost45** [visgl/deck.gl#6261]
#### Target Use case
My use case involves a view for static overlay/interface, and another view for dynamic panning/zooming. Both the overlay view and the dynamic view fill the screen, with the overlay view being on top. The overlay view has controller: false and all contained layers with pickable: false. I am also using event callbacks.

On hover, etc, unless a layer from the dynamic view gets picked, picking will use the static overlay viewport - this means that the generated world coordinate values are projected from the static viewport, instead of the below dynamic viewport, despite having no pickable layers or controller.

#### Proposed feature
Prevent 'overlay' viewports from being selected when picking.

This feature could possibly (?) be implemented in several ways:
1. A boolean prop for overlay views (overlay: true) preventing viewport use in picking
2. Views with controller: false not used for picking

#### To Do List 
  - [ ] Add label and assign to milestone 
  - [ ] Coding
  - [ ] Doc update
  - [ ] What’s new update 
  - [ ] Test

**Comment by Pessimistress** [comment]
Can you try `^8.6.0-alpha`?

**Comment by mcost45** [comment]
@Pessimistress Yep I just tried - top-level Deck hover events are still getting the overlay viewport, but click/drag events are getting the dynamic viewport.

**Comment by mcost45** [comment]
Yes - I'll be able to try that in a week

**Comment by Pessimistress** [comment]
Bug fix (`pickInfo.viewport` populated with different viewports in `onClick` and `onHover` if nothing is picked) published in v8.6.3.

**Comment by mcost45** [comment]
Just checked out the new version in the same codepen. Definitely a nice improvement to have the consistency with all events using the overlay viewport. However it could still be quite nice to have something built in so no overrides are necessary, if it can be done without ambiguity. 

Instead of my previous ideas, possibly there could be an additional view prop - named along the lines of 'redirectPickTo'. This prop could take another viewId, which any events on said view would be redirected to, allowing multi-view projection to easily be implemented as desired. Would this work better/remove the chance for ambiguity?
